Street Trees/STTreeMapViewController.swift

```diff
@@ -82,26 +82,25 @@ class STTreeMapViewController: UIViewController, MKMapViewDelegate, CLLocationMa
     }()
     
     var isArborDay: Bool {
-        let dateComponents = NSCalendar.autoupdatingCurrentCalendar().components([.Month, .Day], fromDate: NSDate())
+        let dateComponents = (Calendar.autoupdatingCurrent as NSCalendar).components([.month, .day], from: Date())
```

Comment:
Pretty sure this was the converter but no need to convert to NSCalendar.

Calendar.autoupdatingCurrent.dateComponents([.day, .month], from: Date())